index.html

```diff
@@ -63,7 +63,7 @@ <h1 class="feature-header">
           <h1 class="feature-header">
             Take a closer look
           </h1>
-          <p>Web Replay's timeline lets you mark points of interest such as exceptions, breakpoints, events, and network requests. </p>
+          <p>Web Replay's timeline highlights points of interest such as exceptions, breakpoints, events, and network requests. </p>
```

Comment:
Can this be "highlights points of interest such as console errors and messages."